PracticeGit/intlist.h

```diff
@@ -0,0 +1,51 @@
+//------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+//---------------------------------------------------------------------------------------------------
+//   intlist.h
+//   Program on Linked list header file
+//--------------------------------------------------------------------------------------------
+/// <summary>
+/// Implementing a linked list data structure to handle integer elements. The implementation will consist of three files:
+///                         intlist.h                -      the header file
+///                         intlist.c                -      the code.
+///                         testintlist.c            -      the program that tests the implementation.
+/// 
+/// The following functions have been implemented:
+/// 
+///    a) Create � this will create a new linked list.
+///    b) Delete � this will delete the list and all the linked elements.
+///	 c) AddAtEnd � this will add an element to the end of the list.
+///    d) Insert � this will insert an element at a particular index(zero based).
+///    e) RemoveAt � this will remove the element at a particular index(zero based).
+///    f) Remove � removes the first occurrence of a specific element.
+///    g) Count � number of elements in the list.
+///    h) Get � gets the element at a particular index.
+///  </	summary>
+//
+// ---------------------------------------------------------------------------------------------
+
+#ifndef INTLIST_H
+#define INTLIST_H
+#define ERROR_MEM_ALLOC -1	  // Return an error if memory allocation failed
+#define ERROR_OUT_BOUNDS -2  // Return an error if index is out of bounds

```

Comment:
ERROR_INDEX_OUT_OF_BOUNDS, the name should be clear and indicative of the problem